_maps/shuttles/pirate_geode.dmm

```diff
@@ -513,7 +527,16 @@
 /area/shuttle/pirate)
 "BI" = (
 /obj/effect/turf_decal/lunar_sand/plating,
-/obj/machinery/computer/launchpad{
+/obj/structure/table/reinforced,
+/obj/item/storage/toolbox/syndicate,
```

Comment:
Undocumented buff?

Reply:
yeah, I got carried away with the changes and forgot to change the desc. i'll document it now